docs/feature_wpm.md

```diff
@@ -10,11 +10,23 @@ For split keyboards using soft serial, the computed WPM score will be available
 
 ## Configuration
 
-|Define                       |Default       | Description                                                                              |
-|-----------------------------|--------------|------------------------------------------------------------------------------------------|
-|`WPM_SMOOTHING`              |`0.0487`      | Sets the smoothing to about 40 keystrokes                                                |
-|`WPM_ESTIMATED_WORD_SIZE`    |`5`           | This is the value used when estimating average word size (for regression and normal use) |
-|`WPM_ALLOW_COUNT_REGRESSOIN` |_Not defined_ | If defined allows the WPM to be decreased when hitting Delete or Backspace               |
+| Define                       | Default       | Description                                                                              |
+|------------------------------|---------------|------------------------------------------------------------------------------------------|
+| `WPM_ESTIMATED_WORD_SIZE`    | `5`           | This is the value used when estimating average word size (for regression and normal use) |
+| `WPM_ALLOW_COUNT_REGRESSOIN` | _Not defined_ | If defined allows the WPM to be decreased when hitting Delete or Backspace               |
```

Comment:
```suggestion
| `WPM_ALLOW_COUNT_REGRESSION` | _Not defined_ | If defined allows the WPM to be decreased when hitting Delete or Backspace               |
```
(the actual code does not have that typo, only the documentation)

Reply:
lol, oops.  That typo was already present in the documentation;  fixing it here to include in this update!